### PR TITLE
[Backend] Auth remove for post contact us Fixed

### LINKED ISSUE #957: [Backend] Auth remove for post contact us
**Describe the bug**
Post contact us doesn't need auth, why because user post the data using contact us. Because of authentication user unable to post the data from contact us page.

**To Reproduce**
1. Go to contact us page
2. enter details and click on let's talk you will see the pop up Missing or invalid authentication header.
3. User data not getting stored.

**Expected behavior**
when user click on let's talk then we have to store the users data.

**Have you read the [Code of Conduct](https://github.com/HITK-TECH-Community/Community-Website/blob/main/code_of_conduct.md)?**
Write your answer here.
yes

### PR DESCRIPTION
**Pull request by Hemu21**
## Issue that this pull request solves

[Issue Link](https://github.com/HITK-TECH-Community/Community-Website/issues/957) resolve #957 
 Closes: #957 

### Brief description of what is fixed or changed
Post contact us doesn't need auth, why because user post the data using contact us. Because of authentication user unable to post the data from contact us page. Now post contact us works fine.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by Hemu21** [comment]
@Kajol-Kumari  when you have a free time please check and merge it.

File: backend/app/routes/contactUs/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const deleteContactUs = require('./delete');
 const { authMiddleware } = require('../../../helpers/middlewares/auth');
 
 router.get('/getcontactus', getContact);
```
Comment (Kajol-Kumari) [inline backend/app/routes/contactUs/index.js:9]: can u add the authMiddleware here as only admins should be able to fetch this data

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Hemu21 as this is for a level2 issue, can you also add middle ware here - https://github.com/HITK-TECH-Community/Community-Website/blob/main/backend/app/routes/joinUs/index.js#L10

the delete and get call should only be accessible to admins

**Comment by Hemu21** [comment]
@Kajol-Kumari updated can you please check.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM

**Pull request merged**